### DISCUSSION TITLE
Day planner - notion - obsidian  synchronize

### DISCUSSION THREAD

**Discussion by fredjoe7** [ivan-lednev/obsidian-day-planner#601] [Q&A] [answered]
i find it easier to do task management in Notion than obsidian....

i would like to integrate time blocking everywhere.

Any ideas to integrate day planner time blocking with notion as well ?

can day planner be synced with todoist app ?

**Comment by ivan-lednev** [comment, marked as answer]
There are currently no integrations with these services. You can create separate feature requests describing how the solution should look like and what value it will give to you and other users.
